.storybook/preview.tsx

```diff
@@ -3,36 +3,44 @@ import '../src/index.scss'
 import { SeamProvider } from '@seamapi/react'
 import type { Preview } from '@storybook/react'
 
-import { fakePublishableKey, fakeUserIdentifierKey } from './seed-fake.js'
-
-const isProd = process.env['NODE_ENV'] === 'production'
-const useFake = process.env['STORYBOOK_SEAM_ENDPOINT'] == null
+const useFake =
+  process.env['DONT_USE_FAKE'] === '1' &&
+  process.env['STORYBOOK_SEAM_ENDPOINT'] == null
 
 const preview: Preview = {
   globalTypes: {
+    /** @deprecated use "some_publishable_key" **/
     publishableKey: {
       description: 'Seam publishable key',
-      defaultValue:
-        process.env['STORYBOOK_SEAM_PUBLISHABLE_KEY'] ?? fakePublishableKey,
+      defaultValue: useFake
+        ? 'some_publishable_key'
+        : process.env['STORYBOOK_SEAM_PUBLISHABLE_KEY'],
     },
+    /** @deprecated use "some_user_identifier_key" **/
     userIdentifierKey: {
       description: 'Seam user identifier key',
-      defaultValue:
-        process.env['STORYBOOK_SEAM_USER_IDENTIFIER_KEY'] ??
-        fakeUserIdentifierKey,
+      defaultValue: useFake
+        ? 'some_user_identifier_key'
+        : process.env['STORYBOOK_SEAM_USER_IDENTIFIER_KEY'],
     },
+    /** @deprecated use "device1" **/
```

Comment:
Why are these marked `deprecated`? Looks like they are already updated to use the new value.

Reply:
i think stories should use the value directly to avoid abstraction so that people don't understand the scenarios.

Reply:
Ah ok, the comments should suggest that though. So maybe they can say something like `Global IDs will depreciated in favor of values local to each story`

Reply:
agree makes more sense